.github/workflows/test-package.yml

```diff
@@ -97,7 +97,6 @@ jobs:
 
     - name: Install requirements
       run: |
-        pip install -r requirements.txt
```

Comment:
Please keep this even if empty, it doesn't hurt and if in a future there will be some new requirement it will already work in the CI.